src/platform/apollolake/include/platform/memory.h

```diff
@@ -152,17 +152,12 @@
 	(HEAP_SYS_RUNTIME_M_SIZE + ((PLATFORM_CORE_COUNT - 1) * \
 	HEAP_SYS_RUNTIME_S_SIZE))
 
-#define HEAP_RUNTIME_BASE \
-	(HEAP_SYS_RUNTIME_0_BASE + HEAP_SYS_RUNTIME_M_SIZE + \
-	((PLATFORM_CORE_COUNT - 1) * HEAP_SYS_RUNTIME_S_SIZE))
-
 #define HEAP_RUNTIME_SIZE \
 	(HEAP_RT_COUNT64 * 64 + HEAP_RT_COUNT128 * 128 + \
 	HEAP_RT_COUNT256 * 256 + HEAP_RT_COUNT512 * 512 + \
 	HEAP_RT_COUNT1024 * 1024)
 
-#define HEAP_BUFFER_BASE	(HEAP_RUNTIME_BASE + HEAP_RUNTIME_SIZE)
-#define HEAP_BUFFER_SIZE	(SOF_STACK_END - HEAP_BUFFER_BASE)
+#define HEAP_BUFFER_SIZE	0x2000
```

Comment:
Buffer heap has only 8kB?

Reply:
Yes, looks like a safe and tested value, @dabekjakub can share more details

Reply:
@abonislawski this looks too small for large buffers or pipelines with multiple buffers. @dabekjakub can you explain the details.

Reply:
@lgirdwood I have asked @wwittbrx for test it on a real platform and it works very well

Reply:
@abonislawski @wwittbrx It should fail for more than 2 pipelines of 48 kHz, 8 channels, 32 bit.

Reply:
issue resolved, just a note for @lgirdwood : @tlauda noticed that there is also used a second buffer HEAP_HP_BUFFER, so no problem with 0x2000 for HEAP_BUFFER_SIZE, tested also with 8 channels@32b